rimage/file_simple.c

```diff
@@ -43,7 +43,7 @@
 #define IMX8M_IRAM_BASE		0x3b6f8000
 #define IMX8M_IRAM_HOST_OFFSET	0x10000
 #define IMX8M_IRAM_SIZE		0x800
-#define IMX8M_DRAM_BASE		0x3b6f8000
+#define IMX8M_DRAM_BASE		0x3b6e8000
```

Comment:
I think we could get both on-DSP DRAM regions by extending the size to 0x10000 on the next line, but I wonder if we should even do that.

Reply:
I don't think we use that area, so we are fine with this change. It is similar with what we do for imx8 and imx8x

Reply:
Fair enough. Boot code may use this area but everything else goes in the SDRAM regions so this is fine.